src/main/resources/templates/pages/featureOverview.vm

```diff
@@ -1,6 +1,9 @@
 <!DOCTYPE html>
 <html xmlns="http://www.w3.org/1999/xhtml">
 <head>
+#if($fromJenkins)
+	<base href="${jenkins_base}job/$build_project/$build_number/cucumber-html-reports/">
```

Comment:
We send the feature report also from jenkins by email. A relative path is not sufficient when you want the user to be able to click right on the (failed) feature.